README.adoc

```diff
@@ -43,18 +43,81 @@ The http://www.hawkular.org/docs/rest/rest-metrics.html[REST API] should be easy
 data with the simplest tools: shell scripts and `curl`.
 
 == Getting started with a release build
+There are a couple of options for running Hawkular Metrics:
 
-http://wildfly.org/downloads/[Download a Wildfly 10] archive and expand it somewhere on your disk.
+* WildFly distribution
+* EAR distribution
 
-Check out the https://github.com/hawkular/hawkular-metrics/releases[Metrics releases] page and download the latest
-version of the web application archive: `hawkular-metrics-api-jaxrs-X.Y.Z.war`
+IMPORTANT: Earlier versions of Hawkular Metrics could be run by deploying `hawkular-metrics-api-jaxrs-X.Y.Z.war`. This
+is longer supported.
 
-Copy this file to the `standalone/deployments` directory of the Wildfly server.
+The WildFly distribution is a pre-configured WildFly 10 server that includes Hawkular Alerts in addition to Hawkular
```

Comment:
I would not mention what version of WF is included because it is packaged. That will make it easy when we upgrade WF versions, we do not need to update the documentation.